distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -150,11 +157,13 @@ type Column
        - op_kind: The kind of binary operator.
        - operand: The right operand to the binary operator.
        - new_name: The name of the resulting column.
-    make_binary_op : Text -> Text -> (Text | Nothing) -> Column
-    make_binary_op self op_kind operand new_name=Nothing =
+       - include_self: If True, then self is used as the first argument;
+         otherwise, it is omitted.
+    make_binary_op : Text -> Text -> (Text | Nothing) -> Boolean -> Column
+    make_binary_op self op_kind operand new_name=Nothing include_self=True =
         effective_new_name = new_name.if_nothing <|
             self.naming_helpers.binary_operation_name op_kind self operand
-        self.make_op op_kind [operand] effective_new_name
+        self.make_op op_kind [operand] effective_new_name include_self
```

Comment:
Does it make sense to add a `make_static_op` and have this as a new method and keep the logic a little simpler?

Reply:
I think that could be a good approach.

OTOH we'd have to duplicate parts of the `make_op` code, but the 2 functions will be simpler.

Maybe we can refactor the common parts into some common helpers? Then we can keep simplicity without duplication. If that seems feasible, I think it would be ideal.

Reply:
Did the make_constant approach.